psi4/src/psi4/libfock/jk.cc

```diff
@@ -71,10 +71,12 @@ JK::~JK() {}
 std::shared_ptr<JK> JK::build_JK(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> auxiliary,
```

Comment:
Changes in this function are related to the CompositeJK front end - probably worth a look.